split_word_bounds.mayhemfile

```diff
@@ -0,0 +1,5 @@
+project: unicode-segmentation
```

Comment:
I actually was referring to renaming the fuzz target itself. This is fine for now, but check your other submissions and see if you can't rename these.